en/hosting_guide/site_creation.rst

```diff
@@ -0,0 +1,106 @@
+Website creation
+================
+
+Creating a new site with Open Orchestra requires three steps. Additionnally, you can add some more
+steps to enhance the site features. Those steps are the following:
+
+* Declaring the website in Open Orchestra
+* Dealing with rights
+* Contributing the home page
+* Creating an Apache virtual host
+* (Optionnal) Configure the 404 & 503 pages
+* (Optional) Generate the robots.txt
+* (Optional) Generate the sitemap.xml
+
+Declaring the site
+------------------
+The first thing to do, is to declare the site in Open Orchestra. To do this there is no need to open
+a file, you only have to log in the Back Office and fill the website creation form.
+
+Full details are available in the user guide, on `website creation page`_.
+
+Dealing with rights
+-------------------
+To be able to contribute to the site, rights have to be set. To do this, create a new group
+including several roles related to this site (including ROLE_ACCESS_TREE_NODE) and assign it to the
+newly created site.
+
+Once this is done, you can add users to the group by editing them. On their next login, these users
```

Comment:
"login" is a username, "log in" is the action. Anyway, I think in this context we should say "their next connection"